lib/FileUpload/useFileUpload.js

```diff
@@ -0,0 +1,35 @@
+import FileUploadService from './FileUploadService';
+
+export const useFileUpload = () => {
+  const fileUploadService = new FileUploadService();
+  const uploadFile = async ({ fields, url }, file) => {
+    if (fields.length > 0 && url) {
+      const [storage, id] = fields.find(({ key }) => key === 'key').value.split('/');
+
+      const formData = new FormData();
+      fields.forEach(({ key, value }) => {
+        formData.append(key, value);
+      });
+      formData.append('file', file);
+
+      try {
+        await fileUploadService.post(url, formData);
```

Comment:
Maybe we could just use native `fetch` here? No reasons to support `axios` in this project.

Reply:
I agree with @Nikita-Polyakov